lib/jekyll-feed.rb

```diff
@@ -12,6 +12,10 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      config["feed_path"] || "feed.xml"
```

Comment:
Ahh. I was thinking the source template. We sanitize prior to the output destination (to prevent overwriting a file outside the site source). 

@parkr is there any problem with it being passed into `keep_files`? Should we sanitize there to be safe?